tests/model_servering/server/llm/conftest.py

```diff
@@ -0,0 +1,177 @@
+import pytest
+from kubernetes.dynamic import DynamicClient
+from kubernetes.dynamic.exceptions import NotFoundError
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.namespace import Namespace
+from ocp_resources.resource import ResourceEditor, get_client
+from ocp_resources.secret import Secret
+from ocp_resources.service_account import ServiceAccount
+from ocp_resources.service_mesh_member_roll import ServiceMeshMemberRoll
+from ocp_resources.serving_runtime import ServingRuntime
+from pytest_testconfig import config as py_config
+
+from tests.model_servering.server.llm.utils import base64_encode_str
+from utilities.constants import TRANSFORMERS_CACHE_ENV_VAR
+from utilities.inference_service import InferenceServiceForTests
+
+
+@pytest.fixture(scope="session")
+def admin_client() -> DynamicClient:
```

Comment:
Shouldn't admin_client be in the general conftest? Right now we only use it here, and we can move it in the next PR, but we can already know we'll need it across the whole repo.

Reply:
it should; it is here just for the sake of this pr. once more tests need it, it should moved up